### PR TITLE
[IT] Add `Beatmap Spotlights/Seasons/*` and `Beatmap Spotlights/Chart Assembly Team`

### PR DESCRIPTION
**Pull request by Sclafus**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap_Spotlights/Chart_Assembly_Team/it.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+tags:
+  - cat
+  - cats
+  - charts assembly team
+---
+
+# Team di montaggio dei grafici
```
Comment (marcostudios) [inline wiki/Beatmap_Spotlights/Chart_Assembly_Team/it.md:8]: "Grafici" is the wrong term, but i can't think of another better term for this.

Comment (marcostudios) [reply wiki/Beatmap_Spotlights/Chart_Assembly_Team/it.md:8]: Wrong as it leads to think of visual graphs

Comment (Sclafus) [reply wiki/Beatmap_Spotlights/Chart_Assembly_Team/it.md:8]: Idk either tbf

Comment (Sclafus) [reply wiki/Beatmap_Spotlights/Chart_Assembly_Team/it.md:8]: Let's fall back to the original name until we find a better alternative

**Review by marcostudios — APPROVED** [review]
LGTM

**Review by TicClick — APPROVED** [review]

**Pull request merged**